omnibus/config/projects/inspec.rb

```diff
@@ -17,21 +17,52 @@
 #
 
 name 'inspec'
+friendly_name "Inspec"
 maintainer 'Chef Software, Inc <maintainers@chef.io>'
 homepage 'https://github.com/chef/inspec'
 
 license 'Apache-2.0'
 license_file '../LICENSE'
 
-# Defaults to C:/inspec on Windows
-# and /opt/inspec on all other platforms
-install_dir "#{default_root}/#{name}"
+# Defaults to C:/opscode/inspec on Windows
+# and /opt/inspec on all other platforms.
+if windows?
+  install_dir "#{default_root}/opscode/#{name}"
+else
+  install_dir "#{default_root}/#{name}"
+end
 
 build_version Omnibus::BuildVersion.semver
 build_iteration 1
 
 dependency 'preparation'
+dependency 'ruby'
+dependency 'rb-readline'
+
 dependency 'inspec'
 
+dependency 'gem-permissions'
+dependency 'shebang-cleanup'
+dependency 'openssl-customization'
+dependency 'version-manifest'
+dependency 'clean-static-libs'
```

Comment:
Clean up steps before the final packaging step.

gem-permissions sets all the gem files permissions accurately in case one of them has a file that isn't world readable (this came from a customer complaint...).
shebang-cleanup ensures that all the scripts in embedded/bin (like bundler or gem or whatever else) explicitly ask for the ruby we install rather than env ruby.
openssl-customization is our set of hacks to inject SSL certs we need to access S3 resources out of the box.
version-manifest writes an log file with all the various software definition versions so that we can easily verify what it is we built.
clean-static-libs goes in and deletes any stray .dll.a and .a files in lib directories because we don't use those for anything after compilation is complete on windows.  It's there to help other native gems link to the binaries we compile... but that's not something we expect people to be doing in the context of inspec.  We don't even give them a compatible compiler to link things with.